lib/libutils/ext/include/util.h

```diff
@@ -47,6 +47,14 @@
 #define ROUNDUP(v, size) (((v) + ((__typeof__(v))(size) - 1)) & \
 			  ~((__typeof__(v))(size) - 1))
 
+#define ROUNDUP_OVERFLOW(v, size, res) (__extension__({ \
+	typeof(*res) __roundup_tmp = 0; \
```

Comment:
missing parentheses ? `typeof(*(res)) __...`

Reply:
You're right, I'll fix.